### ISSUE TITLE
Use our loos T&C's link accessibility contrast on second visit

### ISSUE THREAD

**Issue by jermyk** [neontribe/gbptm#446]
**Describe the bug**
When the user first reads the T&C's link in the Use our Loos page the T&C link is 
clearly visible in blue.
After the link is activated and the user returns to the page it is much less obvious ( possibly failing accessibility standards) for the user.

**To Reproduce**
1. Go to https://gbptm-stage.herokuapp.com/use-our-loos
2. Scroll to the bottom of the page t&c's are clearly visible in blue
3. Click on the link to visit the t&c's
4. Return to https://gbptm-stage.herokuapp.com/use-our-loos
5. t&c's are less visible 

**Desktop**
 - OS: Windows 10: Chrome

**Smartphone**
 - Device: Samsung Android SM J600FN

![Screenshot_103119_032913_PM](https://user-images.githubusercontent.com/4446325/67961137-31a48100-fbf3-11e9-92e9-6bfb90e466b6.jpg)
![Screenshot_103119_032949_PM](https://user-images.githubusercontent.com/4446325/67961199-47b24180-fbf3-11e9-845c-3b3e1d84cd4d.jpg)
![Screenshot_103119_033025_PM](https://user-images.githubusercontent.com/4446325/67961253-5d276b80-fbf3-11e9-826f-2fe386b4c7b6.jpg)
![Screenshot_103119_033100_PM](https://user-images.githubusercontent.com/4446325/67961308-7203ff00-fbf3-11e9-9de1-f20f3e20650d.jpg)
